### DISCUSSION TITLE
Dynamic form

### DISCUSSION THREAD

**Discussion by atlet** [unpoly/unpoly#394] [Q&A] [answered]
I'm creating a master/detail form. It can have multiple lines. I'm adding a new line to form with JQuery ajax call.

![image](https://user-images.githubusercontent.com/1223162/173592153-b0fc49e2-e680-4e07-90d9-ffebf7aa86b1.png)

Problem is, when I submit this form, the data from dynamically created fields doesn't post to server. What I'm missing.

**Comment by adam12** [comment, marked as answer]
Hard to say, and possibly not related to Unpoly.

My first suggestion is to confirm that the new fields are _for sure_ inside the related `<form>` element.

My second suggestion is to confirm that the dynamically added fields are correct. That is, they have a `[name]` attribute, and that attribute is either unique or the `array[]` style. Perhaps the browser only sees one field (or no field, if it is missing a `[name]`).

**Reply by atlet** [reply]
Tnx, fixed. It wasn't Unpoly.
